codebasin/preprocessor.py

```diff
@@ -539,6 +540,11 @@ class ParseError(ValueError):
     """
 
 
+class Visit(Enum):
+    NEXT = (0,)
```

Comment:
Why do you need to have this as a tuple?

Reply:
Good catch, I don't know where this came from. Something like `black` must have introduced it at some point, but  `pre-commit` doesn't complain if I take it out.  I agree it looks weird, so I've changed these back to integers in https://github.com/intel/code-base-investigator/pull/121/commits/72b3d1be820578d38bd669ddff5db86fd6ef0a6a.